### PR TITLE
Standardize e2e naming pages

### PR DESCRIPTION
**Pull request by Hugo-Hontans**

### CODE REVIEW

File: apps/delivery/delivery-e2e/src/integration/CRUD-org.spec.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ beforeEach(() => {
   cy.clearLocalStorage();
   cy.visit('/auth');
   cy.viewport('macbook-15');
-  const p1: LandingPage = new LandingPage();
-  const p2: LoginPage = p1.clickCallToAction();
+  const p1: WelcomeViewPAge = new WelcomeViewPAge();
```
Comment (ioaNikas) [inline apps/delivery/delivery-e2e/src/integration/CRUD-org.spec.ts]: Typo here, WelcomeView**PA**ge

File: apps/delivery/delivery-e2e/src/support/pages/template/TemplateCreatePage.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import NewTemplatePage from './NewTemplatePage';
+import TemplateAddModal from './TemplateAddModal';
```
Comment (ioaNikas) [inline apps/delivery/delivery-e2e/src/support/pages/template/TemplateCreatePage.ts]: Maybe think about renaming it `TemplateTitleForm` as we got the exact same component to add movies. Best thing should be to create a shared component for both case like you did with ConfirmComponent.

I think Deliveries will soon have a name and this component could also be used in that case.

File: libs/organization/src/lib/pages/organization-editable/organization-editable.component.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { startWith, tap } from 'rxjs/operators';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class OrganizationEditableComponent implements OnInit {
+  @HostBinding('attr.page-id') pageId = 'template-editable';
```
Comment (ioaNikas) [inline libs/organization/src/lib/pages/organization-editable/organization-editable.component.ts]: You mean `'organization-editable'`

Comment (Hugo-Hontans) [reply libs/organization/src/lib/pages/organization-editable/organization-editable.component.ts]: The eye of lynx

File: apps/delivery/delivery-e2e/src/support/pages/template/TemplateCreatePage.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/delivery/delivery-e2e/src/support/pages/template/TemplateCreatePage.ts]: I don't understand if you are talking about pages in tests or about components in libs

**Pull request merged**